### PR TITLE
Set up basic client framework with dummy data

### PR DESCRIPTION
**Pull request by anlee3**
@ecuyle @leananepari 
Please review my client set up! Thank you!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "@babel/core": "^7.2.2",
+    "@babel/preset-env": "^7.3.1",
+    "@babel/preset-react": "^7.0.0",
+    "babel-loader": "^8.0.5",
+    "eslint": "^5.13.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-jsx-a11y": "^6.2.1",
+    "eslint-plugin-react": "^7.12.4",
+    "webpack": "^4.29.3",
```
Comment (ecuyle) [inline package.json:31]: Probably want to also install webpack and webpack-cli into prod dependencies as well. You did the right thing by excluding your bundle.js from your .gitignore, but that means when we push our services into our hosting service in production, there will be no bundle.js file for your app to serve. Thus, we will want our prod system to use webpack to build a final bundle.js upon postinstall.

**Review by leananepari — APPROVED** [review]
Looks good to me.

File: package.json (hunk 1 above)
Comment (anlee3) [reply package.json:31]: Great! Thanks for the feedback. I'll fix that.

**Review by ecuyle — APPROVED** [review]
LGTM. Good job!

**Pull request merged**